File: 8kyu/find-multiples-of-a-number/find-multiples-of-a-number.py
# def find_multiples(integer, limit):
#     result = []
#     for i in range(1, limit + 1):
#       if i % integer == 0:
#         result.append(i)
#     return result


def find_multiples(integer, limit):
  return [i for i in range(1, limit + 1) if i % integer == 0]


print(find_multiples(5, 25))

File: 8kyu/find-multiples-of-a-number/find-multiples-of-a-number.ts
export function findMultiples(integer: number, limit: number): number[] {
	const result: number[] = [];
	for (let i = 1; i <= limit; i++) {
		if (i % integer === 0) {
			result.push(i);
		}
	}

	return result;
}

console.log(findMultiples(5, 25));
